src/logo/index.js

```diff
@@ -4,50 +4,55 @@ import React from 'react';
 // Internal Dependencies.
 import './style.scss';
 
-const Logo = ( { className, text, src, alt, onClick, href } ) => {
+const Logo = ( { className, text, src, alt, onClick, href, extraText } ) => {
 	if ( ! text && ! src ) {
 		return '';
 	}
 
 	return (
-		<span
-			className={ `stc-logo ${ className ? className : '' }` }
-			onClick={ ( event ) => {
-				if ( 'function' === typeof onClick ) {
-					onClick( event );
-				}
-			} }
-		>
-			{ src ? (
-				<>
-					{ href ? (
-						<a
-							href={ href }
-							target="_blank"
-							rel="noopener noreferrer"
-						>
+		<>
```

Comment:
```suggestion
```
Since we havn't added any extra span, we can get rid of this wrapper.

Reply:
Thanks done